src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -746,6 +745,36 @@ public File getFiles(String requestId, String format) throws HelloSignException
         return httpClient.withAuth(auth).withGetParam(PARAM_FILE_TYPE_URI, format).get(url).asFile(fileName);
     }
 
+    public String getFilesUrl(String requestId) throws HelloSignException {
+        return getFilesUrl(requestId, SignatureRequest.SIGREQ_FORMAT_PDF);
+    }
+
+    public String getFilesUrl(String requestId, String format) throws HelloSignException {
+        if (format == null || format.isEmpty()) {
+            format = FILES_FILE_EXT;
+        }
+
+        String fileUrl = null;
+        String url = BASE_URI + SIGNATURE_REQUEST_FILES_URI + "/" + requestId;
+
+        HttpClient httpClient = this.httpClient.withAuth(auth).withGetParam(PARAM_FILE_TYPE_URI, format).withGetParam(PARAM_TEMPLATE_GET_URL, "1").get(url);
+
+        if (httpClient.getLastResponseCode() == 404) {
+            throw new HelloSignException(String.format("Could not find request with id=%s", requestId));
```

Comment:
@cmpaul It would be nice to introduce a subclass of `HelloSignException` that might be used to denote a not a resource that was not found